poshc2/server/C2Server.py

```diff
@@ -79,13 +84,18 @@ def do_GET(self):
             response_content_type = "text/html"  
             response_content = None
             
+
+            # if (now - last_cached) > 30s:
```

Comment:
We adding caching later? If not adding at all we can remove the variables at the top.

Reply:
adding later